controller/loginController.js

```diff
@@ -0,0 +1,113 @@
+const User = require('../models/user')
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt')
+const http = require('http-status-codes')
+const secretKey = require('../config/secretKey')
+const statusMsg = require('../config/statusMsg')
+const dotenv = require('dotenv')
+dotenv.config({
+    path: './config/.env'
+})
+const tokenGenerator = require('./authTokenGenerator')
+
+exports.login = async (req, res, next) => {
+    try {
+        if (req.body.email === '' || req.body.password === '') {
+            let err = new Error()
+            err.status = http.CONFLICT
+            next(err)
+        }
+        let user = await User.findOne({ email: req.body.email })
+        if (!user) {
+            let err = new Error()
+            err.status = http.BAD_REQUEST
+            return next(err)
```

Comment:
@AwaleRohin if you are using the error handler as middleware only you will be able to use ```next``` method

Reply:
yes, i am using error handler as middleware